src/groups/translation_group.jl

```diff
@@ -0,0 +1,17 @@
+@doc doc"""
+    TranslationGroup{N} <: GroupManifold{Euclidean{N},AdditionOperation}
+
+Translation group `\mathrm{R}(N)` represented by translation arrays.
```

Comment:
I prefer the notations `\mathrm{T}(N)`, `\mathrm{T}^N`, or `\mathbb{R}^N`

Reply:
I'll change it to `\mathrm{T}(N)` then.